compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/WindowExceptionHandlerFactory.desktop.kt

```diff
@@ -0,0 +1,37 @@
+package androidx.compose.ui.awt
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.ExperimentalComposeUiApi
+import java.awt.Window
+import java.awt.event.WindowEvent
+import javax.swing.JFrame
+import javax.swing.JOptionPane
+import javax.swing.SwingUtilities
+
+/**
+ * Default [WindowExceptionHandlerFactory], which will show standard error dialog, and close the window after that
+ */
+@ExperimentalComposeUiApi
+object DefaultWindowExceptionHandleFactory : WindowExceptionHandlerFactory {
+    override fun exceptionHandler(window: Window) = Thread.UncaughtExceptionHandler { _, throwable ->
```

Comment:
Why use `Thread.UncaughtExceptionHandler`. Maybe better design this interface being MPP-ready? I.e. just return a lambda.

Reply:
Let's do it MPP. Can't move it though for now, until we make MPP Window